README.md

```diff
@@ -40,90 +40,34 @@ An easy way to get the full Panoptes stack running (see `fig.yml` to dig into th
   * [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) - Docker
   * [Windows](http://docs.docker.com/installation/windows/) - Boot2Docker
 
-#### Installation
+* [fig](http://fig.sh)
 
-1. Ensure your repo directory starts with a lowercase letter - you may need to move `/Panoptes` to `/panoptes`.
+#### Usage
 
-2. Prepare your fig development environment config files. You should only have to do this before the first boot. **Note:** The fig docker environment uses linked docker containers, so your Postgres and Zookeeper hosts urls need to refer to these containers.
+1. Run `rake configure FIG_RAKE=off`
```

Comment:
Should we put more context about what these steps are trying to achieve. E.g. "1. Setup the application configuration files, run `rake configure FIG_RAKE=off`"

And likewise for steps 2 and 3.